examples/feature_selection/plot_feature_selection.py

```diff
@@ -76,8 +89,7 @@
 svm_weights = np.abs(clf[-1].coef_).sum(axis=0)
 svm_weights /= svm_weights.sum()
 
-plt.bar(X_indices - 0.25, svm_weights, width=0.2, label="SVM weight")
-
+# After univariate feature selection
```

Comment:
Instead, to add a comment, you can add a separator and print only the result for this part.
Print the accuracy for the model above.